R/conversion.R

```diff
@@ -160,7 +160,29 @@ py_to_r.pandas.core.series.Series <- function(x) {
   py_to_r(x$as_matrix())
 }
 
+#' @export
+summary.pandas.core.series.Series <- function(object, ...) {
+  if (py_is_null_xptr(object) || !py_available())
+    str(object)
+  else
+    object$describe()
+}
+
+#' @export
+length.pandas.core.series.Series <- function(x) {
+  if (py_is_null_xptr(x) || !py_available())
+    0L
+  else
+    py_to_r(x$size)
```

Comment:
You might want to use `as_r_value(x$size)` since this might auto-convert depending on how the Series object was constructed.